GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift

```diff
@@ -219,17 +217,23 @@ class ChatMessageEntryView: BaseView {
             height: CGFloat.greatestFiniteMagnitude
         )
 
-        // We need to take into account larger height
-        // of text view or placeholder label for
-        // updated text entry height.
-        var newHeight = max(textView.sizeThatFits(size).height, placeholderLabel.sizeThatFits(size).height)
+        guard let font = textView.font else {
```

Comment:
do we need to handle here not valid lineHeight that can lead to division by zero error?
```suggestion
        guard let font = textView.font, font.lineHeight > 0 else {
```